scripts/experiments_tabular/maf_run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#SBATCH --time=24:00:00
+#SBATCH --nodes=1 --ntasks-per-node=4 --cpus-per-task=1
+#SBATCH --mem=16G
+#SBATCH --qos==collaborator
+
+MAX_SEED=$1
+DATASET=$2
+
+module load Julia/1.5.1-linux-x86_64
+module load Python/3.8.2-GCCcore-9.3.0
```

Comment:
Do you still need Python and py-env?

Reply:
Unfortunatelly yes. I may be wrong, but once you import PyCall, it always needs some python to call home during startup. If we went the miniconda route, we would not have to deal with it.